classes/system/stacklight/server/cluster.yml

```diff
@@ -7,3 +8,10 @@ classes:
 parameters:
   _param:
     kibana_elasticsearch_host: ${_param:monitor_vip_address}
+  keepalived:
+    cluster:
+      instance:
+        VIP:
```

Comment:
nit but can you use a more explicit name (eg 'monitoring_vip' or 'stacklight_vip') because eventually it will be used for tagging the vrrp metrics.